Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/eventregistration/ui/createevent/CreateEventTestViewModel.kt

```diff
@@ -32,38 +35,51 @@ class CreateEventTestViewModel @AssistedInject constructor(
         address: String,
         start: String,
         end: String,
-        defaultCheckInLengthInMinutes: String
+        defaultCheckInLengthInMinutes: String,
+        sendToServer: Boolean = false
     ) {
         try {
             val startDate =
                 if (start.isBlank()) null else DateTime.parse(start, DateTimeFormat.forPattern("yyyy-MM-dd HH:mm"))
             val endDate =
                 if (end.isBlank()) null else DateTime.parse(end, DateTimeFormat.forPattern("yyyy-MM-dd HH:mm"))
 
-            /* TODO: wait for new protobuf messages 'TraceLocation' and perform network request to get
-                'SignedTraceLocation' */
-
-            // Backend needs UNIX timestamp in Seconds, not milliseconds
-            val startTimeStampSeconds = startDate?.toInstant()?.seconds ?: 0
-            val endTimeStampSeconds = endDate?.toInstant()?.seconds ?: 0
-
             val traceLocationType =
-                if (type == "Event") TraceLocationOuterClass.TraceLocationType.LOCATION_TYPE_TEMPORARY_OTHER
-                else TraceLocationOuterClass.TraceLocationType.LOCATION_TYPE_PERMANENT_OTHER
+                if (type == "Event") LOCATION_TYPE_TEMPORARY_OTHER else LOCATION_TYPE_PERMANENT_OTHER
 
-            val traceLocation = TraceLocation(
-                UUID.randomUUID().toString(), // will be provided by the server when the endpoint is ready
-                traceLocationType,
-                description,
-                address,
-                startDate?.toInstant(),
-                endDate?.toInstant(),
-                defaultCheckInLengthInMinutes.toInt(),
-                "ServerSignature".toByteArray().toByteString()
-            )
+            if (sendToServer) {
+                val userInput = TraceLocationUserInput(
```

Comment:
Could you add some default values to the test fragment option for creating events, to make it easier (faster) to send a test event?

Reply:
Really good idea! I added default values!